protobuf.spec

```diff
@@ -8,25 +8,30 @@
 # protoc --cpp_out=. DQMServices/Core/src/ROOTFilePB.proto
 #######################################################
 
-Source: https://github.com/google/protobuf/archive/v%{realversion}.tar.gz
-Patch0: protobuf-3.15-gcc10
+%define keep_archives true
+
+Source: https://github.com/protocolbuffers/protobuf/archive/v3.21.9.zip
+#Patch0: protobuf-3.15-gcc10
+#Patch1: protobuf-std_iterator-8741
 Requires: zlib
 BuildRequires: cmake ninja
 
 %prep
 %setup -n %{n}-%{realversion}
-%patch0 -p1
-sed -i -e 's|CMAKE_CXX_STANDARD  *11|CMAKE_CXX_STANDARD 17|' cmake/CMakeLists.txt
+#%patch0 -p1
+#%patch1 -p1
+#sed -i -e 's|CMAKE_CXX_STANDARD  *11|CMAKE_CXX_STANDARD 17|' cmake/CMakeLists.txt
```

Comment:
@iarspider , why we are not changing the cxx standard? I think default is still `11` ... right?

Reply:
ping